modules/Bio/EnsEMBL/Compara/PipeConfig/BaseAge_conf.pm

```diff
@@ -59,19 +61,29 @@ sub default_options {
 #            'name' => "node_id",
             'name' => "name",
 
-            #Location url of database to get EPO GenomicAlignTree objects from
-            #'compara_db' => 'mysql://ensro@compara3:3306/cc21_mammals_epo_pt3_86',
+            # Location url/alias of database to get EPO GenomicAlignTree objects from
+            'compara_db'   => 'compara_curr',
+            'ancestral_db' => 'ancestral_curr',    # You may set this to undef if compara_db is a database that contains genome_db locators
 
-            #Location url of database to get snps from
-            #'variation_url' => 'mysql://ensro@ens-staging1:3306/homo_sapiens_variation_86_38?group=variation',
+            # The name of the alignment
+            'species_set_name' => undef,
+
+            # There is a different colour gradient for this clade
+            'clade_taxon_id' => undef,
+
+            # Location url of database to get snps from
+            #'variation_url' => 'mysql://anonymous@mysql-ensembl-mirror:4240/' . $self->o('ensembl_release')
+            'variation_url' => undef,
```

Comment:
Ah `variation_url` is still there because we haven't yet merged _release/99_ into _master_. Nevermind, I'll do the merge and update this later